### PR TITLE
add 'or equivalent algo' text

### LINKED ISSUE #605: Add to top of Appendix B "or any equivalent algorithm"

### PR DESCRIPTION
**Pull request by fluffy**
Fixes #605

### CODE REVIEW

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -6283,6 +6283,13 @@ a=msid:bbce3ba6-abfc-ac63-d00a-e15b286f8fce
       Description" of <xref
       target="I-D.ietf-mmusic-sdp-bundle-negotiation" />.</t>
 
+      <t> Applications can implement RTP stacks in many different
+      ways. The text in this section is not meant to be prescriptive
+      about how an RTP stack is implemented. Applications MAY use any
+      algorithm or implementations that archives equivalent results to
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: achieves

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -6283,6 +6283,13 @@ a=msid:bbce3ba6-abfc-ac63-d00a-e15b286f8fce
...
+      an implementations that works according the description in this
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: I would say "to those described by this algorithm"

File: draft-ietf-rtcweb-jsep.xml (hunk 1 above)
Comment (fluffy) [inline draft-ietf-rtcweb-jsep.xml]: fixed

File: draft-ietf-rtcweb-jsep.xml (hunk 2 above)
Comment (fluffy) [inline draft-ietf-rtcweb-jsep.xml]: fixed

**Pull request merged**